resources/kiali/values.yaml

```diff
@@ -27,11 +27,13 @@ global:
     enabled: true
 
 kcproxy:
+  enabled: true
   replicaCount: 1
   port: 3000
   config:
     clientId: "kiali"
     clientSecret: "hiFWLWqIxw5d3gl"
+    discoveryUrl: https://dex.{{ .Values.global.ingress.domainName }}
```

Comment:
Hmm, what is the reason for introducing this property?

Reply:
Icke asked me to introduce it so that if this URL needs to be adapted, everything can be done in the 'values.yaml'. 
The property gets used in here: https://github.com/kyma-project/kyma/pull/10813/files#diff-3970e9005cb9dd2ffec95600b78d8362922633355d029c151838d6620858e6afR36

The same goes for the tracing charts.

Reply:
Thanks, I see!

@a-thaler Is it something we want to make overridable? Because if it's just because of code reuse, we could just put it into a template (a *.tpl file)

Reply:
It was about having the OIDC issuer related settings configurable. In future there might be no dex and you can enable it with your custom settings